api/primary.go

```diff
@@ -6,6 +6,7 @@ import (
 
 	log "github.com/Sirupsen/logrus"
 	"github.com/docker/swarm/cluster"
+	"github.com/docker/swarm/pkg/authZ"
```

Comment:
It's better to `godep` this package rather than putting it directly under `pkg/authZ`.